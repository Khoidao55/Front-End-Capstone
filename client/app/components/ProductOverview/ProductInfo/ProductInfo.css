.hello {
  display: flex;
}

.product-item-summary {
  width: 1000px;
  border-right: solid;
  padding-right: 30px;
}

.product-features {
  display: flex;
  flex-direction: column;
  margin-left: 30px;
}

.product-feature-single {
  display: flex;
}

.feature-check {
  margin-top: 15px;
  margin-right: 20px;
}