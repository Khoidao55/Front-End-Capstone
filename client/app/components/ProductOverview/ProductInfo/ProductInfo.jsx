import React from 'react';
import './ProductInfo.css';

const ProductInfo = ({ product }) => {
  return (
    <div className="product-info-innerContainer">
      <div className="product-item-summary">
        <h2>{product.slogan}</h2>
        <p>{product.description}</p>
        <div>
          <p>Share: </p>
          <i style={{cursor: 'pointer', fontSize: '30px'}} className="fab fa-twitter-square"></i>
          <i style={{cursor: 'pointer', paddingLeft: '10px', fontSize: '30px'}} className="fab fa-facebook-square"></i>
          <i style={{cursor: 'pointer', paddingLeft: '10px', fontSize: '30px'}} className="fab fa-pinterest-square"></i>
        </div>
      </div>
      <div className="product-features">
        {product.features.map((itemFeature) => {
          return (
            <div className="product-feature-single">
              <i className="feature-check fas fa-check"></i>
              <p>{itemFeature.feature}, {itemFeature.value}</p>
            </div>
          )
        }) }
      </div>
    </div>
  )
};

export default ProductInfo;
