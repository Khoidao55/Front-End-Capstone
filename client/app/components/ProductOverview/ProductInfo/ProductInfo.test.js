import React from 'react';
import { shallow } from 'enzyme';
import ProductInfo from './ProductInfo.jsx';

const dummyProps = {
  product: {
    slogan: 'cool slogan',
    description: 'cool description',
    features: [],
  },
};

describe('Product Info', () => {
  test('should render my component', () => {
    const wrapper = shallow(<ProductInfo {...dummyProps} />);

    expect(wrapper).toMatchSnapshot();
  });
});
