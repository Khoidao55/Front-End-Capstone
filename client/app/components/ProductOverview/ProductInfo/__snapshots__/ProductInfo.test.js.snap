// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Info should render my component 1`] = `ShallowWrapper {}`;
